### PR TITLE
Fixed tag mistach at open.mp

### PR DESCRIPTION
**Pull request by 21thyear**

### CODE REVIEW

**Comment by ziggi** [comment]
This will result in a tag mismatch for SA-MP I guess

File: foreach.inc [hunk 1]
```diff
@@ -1021,7 +1021,11 @@ stock Iter_ScriptInit()
 		Iter_CreateVehicle
 	*/
 
+#if defined _INC_open_mp
+	stock Iter_CreateVehicle(modelid, Float:x, Float:y, Float:z, Float:angle, color1, color2, respawn_delay, bool: addsiren = false)
```
Comment (ziggi) [inline foreach.inc:1025]: `bool: addsiren` - excess space

**Review by ziggi — CHANGES_REQUESTED** [review]
Now it's better

File: foreach.inc (hunk 1 above)
Comment (21thyear) [reply foreach.inc:1025]: Yes, it's better

Comment (ziggi) [reply foreach.inc:1025]: Can you remove that excess space symbol?

Comment (21thyear) [reply foreach.inc]: Done

**Comment by punkochel** [comment]
@ziggi, @21thyear, maybe it's better like this:
```pawn
#if define _INC_open_mp
#   define FOREACH_TAG_BOOL bool
#   define FOREACH_TRUE true
#   define FOREACH_FALSE false
#else
#   define FOREACH_TAG_BOOL _
#   define FOREACH_TRUE 1
#   define FOREACH_FALSE 0
#endif

stock Iter_CreateVehicle(modelid, Float:x, Float:y, Float:z, Float:angle, 
                        color1, color2, respawn_delay,
                        FOREACH_TAG_BOOL:addsiren = FOREACH_FALSE)
{

}

**Comment by ziggi** [comment]
@punkochel it looks ugly in both cases, but I prefer @21thyear version.

**Comment by 21thyear** [comment]
No better 😁

**Pull request merged**